cibuildwheel/windows.py

```diff
@@ -101,7 +106,10 @@ def install_pypy(version, arch, url):
     return installation_path
 
 
-def build(project_dir, output_dir, test_command, test_requires, test_extras, before_build, build_verbosity, build_selector, repair_command, environment):
+def build(project_dir: str, output_dir: str, build_selector: Callable[[str], bool],
```

Comment:
build_selector should be a BuildSelector object, I think.

Reply:
If you prefer, yes. I made it this way, such the code here (and in `macos.py` and `linux.py`) is agnostic about the actual type (and doesn't need to import it), but just knows it need to be able to be called.